### PR TITLE
its oranges

### PR DESCRIPTION
**Pull request by Cheridan**

### CODE REVIEW

**Comment by Zilenan91** [comment]
oranges way of thanking you for this is to open a pr about retard speech and spam open and close on it

**Comment by Buggy123** [comment]
Shouldn't those all be uppercase anyway?

**Comment by Cheridan** [comment]
@Buggy123 no
it's oranges

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -1196,7 +1196,7 @@
 	desc = "Sour seeds."
 	icon_state = "seed-orange"
 	species = "orange"
-	plantname = "Orange Tree"
+	plantname = "orange Tree"
```
Comment (AndrewJacksonThe2nd) [inline code/modules/hydroponics/seeds.dm]: this actually needs to stay uppercase for consistency on Plant Scanners

**Comment by Core0verload** [comment]
@cheridan, r u still drunk?

**Comment by Cheridan** [comment]
yes

**Comment by Cheridan** [comment]
it's a joke, guys. c'mon now.

**Comment by PKPenguin321** [comment]
ive been pranked

**Pull request closed without merging**